### PR TITLE
trail

### PR DESCRIPTION
**Pull request by Adityarauthan2409**
fixed issue

### CODE REVIEW

File: src/data/contributors/Aditya.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Aditya Rauthan",
+  "skills": ["Python", "Django", "Bootstrap"],
+  "cause": "tech advancement",
+  "image": "Avatar3",
+  "github": "https://github.com/Adityarauthan2409",
+  
+  "linkedin": "https://www.linkedin.com/in/aditya-rauthan"
```
Comment (Abhishek-kumar09) [inline src/data/contributors/Aditya.json:8]: Hey @Adityarauthan2409 , Seems like you have messed up with your linked profile link.
image
The url pointing to the profile is not available. please correct it.

**Review by Suyash-Purwar — COMMENTED** [review]
This suggestion is made to shed light on the pending PRs for a long time. I don't see any activity on his profile and I think the contributor has abandoned his intended contribution in the middle. So, it would be better to just take suggested actions on this and keep the number of pending PRs low.

**Comment by Abhishek-kumar09** [comment]
Thanks @Suyash-Purwar for your detailed review. Looks like the contributor is not in contact for a long time.
I will change the link and merge it for now.

**Comment by Suyash-Purwar** [comment]
Glad to know I was of help! Looking forward to contribute further. 😀

**Pull request merged**